samples/drivers/led_strip/src/main.c

```diff
@@ -31,7 +39,7 @@ static const struct led_rgb colors[] = {
 	RGB(0x00, 0x00, 0x0f), /* blue */
 };
 
-struct led_rgb pixels[STRIP_NUM_PIXELS];
+static struct led_rgb pixels[STRIP_NUM_PIXELS];
 
 static const struct device *const strip = DEVICE_DT_GET(STRIP_NODE);
 
```

Comment:
While you are at it, please remove the new line below (at line 63), and eventually add one the `led_strip_update_rgb()` call.

And eventually shortened the `color` computation to a single line.:

`color = (color + 1) % ARRAY_SIZE(colors);`

Reply:
Improved loop handling to make it easier to understand.

Reply:
It is much better, thanks !